.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -8,11 +8,22 @@ assignees: ''
 ---
 
 **Concisely describe the proposed feature**
-A clear and concise description of what you want. For example, 
-> I would like to add an Apple Metal backend to the compiler so that my Macbook GPU can be utilized.
+A clear and concise description of what you want. For example:
+
+I would like to add an Apple Metal backend to the compiler so that my Macbook GPU can be utilized.
+
+**Think about the advantages and disadvantages of the proposed feature**
+All coins have its two side, we encourage people to think critically. For example:
+
+Advantages:
+1. So that Apple users could utilize their GPU resources.
+
+Disadvantages:
+1. Increases the maintainance cost over multiple backends.
 
 **Describe the solution you'd like (if any)**
 A clear and concise description of what you want to achieve and implement.
+You may also describe an alternative solution you've thought (if any).
```

Comment:
```suggestion
```

Sorry but I don't understand this - do you want the issue opener to describe a second solution?

Reply:
Yes, better expressions?